src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
-import * as serviceWorker from './serviceWorker';
+import FirebaseContext from './Server/context';
+import fb from './lib/firebase';
 
-ReactDOM.render(<App />, document.getElementById('root'));
-
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister();
+ReactDOM.render(
+  <FirebaseContext.Provider value={fb}>
```

Comment:
I suggest you to use what's provided by the library, that way we will simplify all the development process in future iterations.

That said, you don't need to create your own FirebaseContext.

![](https://cdn.zappy.app/d1c5bfef1bc9934fea3304e8800947b9.png)